.editorconfig

```diff
@@ -0,0 +1,17 @@
+# Tab indentation for JS files
+[*.js]
+indent_style = space
+indent_size = 2
+
+[*.ts]
+indent_style = space
+indent_size = 2
+
+[*.md]
+indent_style = space
+indent_size = 2
+
+
+[*.graphql]
+indent_style = space
+indent_size = 2
```

Comment:
Shall we remove this file?

Reply:
Good one, I will try. I had issues with the .md file and copied it from core. Do prettier and eslint work for MB files?